### PR TITLE
Update make.jl

### PR DESCRIPTION
**Pull request by jd-lara**

### CODE REVIEW

**Comment by jd-lara** [comment]
@daniel-thom this fixes the docs failures and updates documenter to v1

**Comment by jd-lara** [comment]
@daniel-thom I opened this https://github.com/JuliaLang/Pkg.jl/issues/3937 

I can't understand how this issue happens or how to reproduce. We only see this in the IS documentation

**Comment by jd-lara** [comment]
> @daniel-thom I opened this [JuliaLang/Pkg.jl#3937](https://github.com/JuliaLang/Pkg.jl/issues/3937)
> 
> I can't understand how this issue happens or how to reproduce. We only see this in the IS documentation

In other news, I fixed this by making a change to use latest Julia for documentation instead of 1.6

**Comment by jd-lara** [comment]
https://nrel-sienna.github.io/InfrastructureSystems.jl/previews/PR384/

File: docs/src/dev_guide/tests.md [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
+
 While developing code and tests it can be convenient to run a subset of tests.
 You can do this with a combination of TestEnv.jl and ReTest.jl.
 
 **Note**: Per recommendations from the developers of TestEnv.jl, install the package
 in your global julia environment. Do the same for Revise.jl.
 
 ```
-$ julia
+\$ julia
```
Comment (daniel-thom) [inline docs/src/dev_guide/tests.md:20]: What are the backslashes for?

Comment (jd-lara) [reply docs/src/dev_guide/tests.md:20]: The documenter throws an error if there are not escaped $ symbols

Comment (jd-lara) [reply docs/src/dev_guide/tests.md]: I reverted the change

**Review by daniel-thom — APPROVED** [review]

**Pull request merged**